src/libmain.c

```diff
@@ -1231,6 +1230,9 @@ gint main_lib(gint argc, gchar **argv)
 #endif
 
 	gtk_main();
+#ifdef G_OS_WIN32
+	win32_free_argv_made_in_utf8(argc, argv);
```

Comment:
Maybe the call including the G_OS_WIN32 guard could go into a seperate function in src/libmain.c as it should be also in https://github.com/geany/geany/blob/master/src/libmain.c#L1090.
Strictly there are more places where it should be called, e.g. every place where we call `exit()` but this would require some refactoring before which is out of scope of this PR (and technically it is not very necessary as the memory in cleaned up anyway on `exit()`).